### PR TITLE
[Neo Benchmark: Fix] fix the benchmark

### PR DESCRIPTION
**Pull request by Jim8y**
# Description

Since we have updated the default value of the protocol setting, it actually breaks the existing benchmark. This pr trys to fix it.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

**Test Configuration**:

Run the benchmark in the debug mode.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: benchmarks/Neo.Benchmarks/config.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
+{
+  "ApplicationConfiguration": {
```
Comment (cschuchardt88) [inline benchmarks/Neo.Benchmarks/config.json:2]: Is this config still rquired now that you're using the `ProtocolSettings.Default`?

Comment (Jim8y) [reply benchmarks/Neo.Benchmarks/config.json:2]: yes

**Comment by Jim8y** [comment]
@superboyiii :

Neo -> D:\a\neo\neo\src\Neo\bin\Debug\net7.0\Neo.dll
  Neo.UnitTests -> D:\a\neo\neo\tests\Neo.UnitTests\bin\Debug\net7.0\Neo.UnitTests.dll
Test run for D:\a\neo\neo\tests\Neo.UnitTests\bin\Debug\net7.0\Neo.UnitTests.dll (.NETCoreApp,Version=v7.0)
Microsoft (R) Test Execution Command Line Tool Version 17.7.2 (x64)
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
A total of 1 test files matched the specified pattern.
  Failed TestCreateAccount [30 ms]
  Error Message:
   Assert.IsTrue failed. 
  Stack Trace:
     at Neo.UnitTests.Wallets.NEP6.UT_NEP6Wallet.TestCreateAccount() in D:\a\neo\neo\tests\Neo.UnitTests\Wallets\NEP6\UT_NEP6Wallet.cs:line 94
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodInvoker.Invoke(Object obj, IntPtr* args, BindingFlags invokeAttr)

  Neo.CLI -> D:\a\neo\neo\src\Neo.CLI\bin\Debug\net7.0\neo-cli.dll
  Neo.GUI -> D:\a\neo\neo\src\Neo.GUI\bin\Debug\net7.0-windows\Neo.GUI.dll

Failed!  - Failed:     1, Passed:   [68](https://github.com/neo-project/neo/actions/runs/7658303261/job/20870922967?pr=3109#step:5:69)6, Skipped:     0, Total:   687, Duration: 17 s - Neo.UnitTests.dll (net7.0)


this test error appears randomly. Can you please check?

**Comment by superboyiii** [comment]
> this test error appears randomly. Can you please check?
At leat this case always runs successfully on mine. But I notice if run all UT cases, some parameters will pollute other cases which could cause mismatching the expected result. But if run these failed cases independently, they will work well. Maybe it's better to fix these later.

`run all`:


`re-run the failed`:
image

**Review by shargon — APPROVED** [review]

**Pull request merged**